### DISCUSSION TITLE
Admin Portal login details

### DISCUSSION THREAD

**Discussion by ddevadat** [Sunbird-RC/community#739] [Questions (FAQs)] [answered]
I have created a core registry using docker compose and able to create entities. Eg. i created an institiute using api invocation.

Now i started up the admin portal and able to login with user created for institute. But with this user i am not able to create more schema.

Is there any admin user available for admin portal

**Comment by Pratikshakhandagale** [comment, marked as answer]
@ddevadat Please use below mentioned login credentials
`admin3`
